_includes/footer.html

```diff
@@ -18,7 +18,7 @@
 <footer class="print-footer" style="display: none;">
   <div class="row">
     <div class="one-half first">
-      <img src="https://cdn.bibleineverylanguage.org/wp-content/uploads/wacs_logo.png" alt="BTT"/>
+      <img src="https://cdn.bibleineverylanguage.org/wp-content/uploads/2018/09/wacs_logo.png" alt="BTT"/>
```

Comment:
Code changes should be in a separate PR from build stuff, but this might not be worth changing.

Reply:
fair. I think it's minor enough we can leave it.